src/adapters/rw-adapter/src/index.ts

```diff
@@ -348,11 +349,12 @@ export default class RwAdapter implements Adapter.Service {
 
     return filters
       .filter(filter => filter.value !== undefined && filter.value !== null)
-      .map(({ column, type, value, notNull }) => ({
-        name: column,
-        type,
-        value: getSerializedValue(value, type),
-        notNull,
+      .map(filter => ({
+        name: filter.column,
+        type: filter.type,
+        operation: filter.operation,
```

Comment:
I actually forgot to serialise the `operation`. The types helped me catch the error 😅!